vocab/properties.json

```diff
@@ -584,22 +578,22 @@
         ]
     },
     "email": {
-        "description": null,
+        "description": "Means or system for transmitting messages electronically.",
         "name": "Email",
         "sameAs": null,
         "schemas": [
             "core/schemas/v3/actors/contactInformation.schema.tpl.json"
         ]
     },
     "endDate": {
-        "description": null,
+        "description": "Point at which something terminates in time",
```

Comment:
if it's a date, it's not a point in time. How about "Date on which something terminates"?

Reply:
I see your point. I was trying to follow the rule that the description should explain the terms used for the property and not reuse those terms in the explanation, but in some cases this might be too much and difficult to still be accurate. I'll change it to your suggestion.